homeassistant/components/neptune_apex/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""The Neptune Apex integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+import async_timeout
+from pynepsys import Apex
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
We don't need this. Please remove the `CONFIG_SCHEMA`.

Reply:
OK